charts/osm/templates/fluentbit-configmap.yaml

```diff
@@ -21,10 +21,10 @@ data:
       match      *
       regex       message /"level":"error"/
     [OUTPUT]
-      Name    {{ .Values.OpenServiceMesh.fluentBit.defaultOutput }}
+      Name    {{ .Values.OpenServiceMesh.fluentBit.outputPlugin }}
       Match   *
-      {{- if not .Values.OpenServiceMesh.fluentBit.allowCustomOutput }}
-      Customer_ID {{ .Values.OpenServiceMesh.fluentBit.workspaceID }}
+      {{- if eq .Values.OpenServiceMesh.fluentBit.outputPlugin "azure" "Azure" }}
```

Comment:
you may want to consider making the value lowercase before comparing to "azure"
via the `lower` function defined in the [Sprig library](http://masterminds.github.io/sprig/strings.html) which is used to give Helm charts more templating functions.